fades/file_options.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2014-2016 Facundo Batista, Nicolás Demarchi
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 3, as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranties of
+# MERCHANTABILITY, SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR
+# PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# For further info, check  https://github.com/PyAr/fades
+
+"""Parse fades options from confir files."""
+
+import logging
+from configparser import ConfigParser
+
+logger = logging.getLogger(__name__)
+
+CONFIG_FILES = ("/etc/fades.ini", "~/.fades.ini", ".fades")
+
+MERGEABLE_CONFIGS = ("dependency", "pip_options", "virtualenv-options")
+
+
+def get_parsers(config_files=CONFIG_FILES):
+    """
+    Search for config files return a ConfigParser for each one.
+
+    Return a Configparser with all files merged in the first position and one
+    Configparser for each file ordered by priority.
+    """
+    parser = ConfigParser()
+    existing_files = parser.read(config_files, encoding='cp1250')
+    if existing_files and parser.sections():
+        if len(existing_files) == 1:
+            return [parser]
+        parsers = []
+        parsers.append(parser)  # first parser is the one with all the settings.
+        for config_file in existing_files:
+            single_parser = ConfigParser()
+            single_parser.read(config_file, encoding='cp1250')
```

Comment:
Aren't we parsing the files *twice*? Why not directly parse the config_files one by one and return them?